tests/mockers.py

```diff
@@ -353,6 +353,8 @@ def launch_command(*args: str) -> str:
 
 def assert_command(cmds: Iterable[str], expected_result: str, strip_indentation: bool = True, indent: str = '  ') -> None:
     expected_result = adapt(expected_result, indent) if strip_indentation else expected_result
+    e = expected_result
+    l = launch_command(*cmds)
     assert launch_command(*cmds) == expected_result
```

Comment:
Hmm `assert l == e`?

Reply:
forgot to remove this, thx

Reply:
Actually... was the reason why you've done has something to do with https://github.com/VirtusLab/git-machete/issues/603 ?